app/src/lib/stores/git-store.ts

```diff
@@ -910,8 +910,6 @@ export class GitStore extends BaseStore {
       this._tip = { kind: TipState.Unknown }
     }
 
-    this.emitUpdate()
```

Comment:
This change makes me nervous. I don't think we can clearly foresee the consequences of not emitting here. This method is from several places in the code base and I'd rather not alter its behaviour.

Reply:
I had a hesitation as well while deleting this line of code, I will check if there is another way...